cmd/internal/cli/x509.go

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2022, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
```

Comment:
If possible, this implementation should not be in the `cmd/internal/cli` package but somewhere appropriate under `internal/pkg`.

We're trying to move implementation logic out of the `cli` package, so that it really just has flag / option handling etc.

Reply:
I see. It shouldn't be a problem.